kordac/Kordac.py

```diff
@@ -2,22 +2,35 @@
 import mdx_math
 from kordac.KordacExtension import KordacExtension
 
-class Kordac():
+class Kordac(object):
+    """A converter object for converting markdown
+    to HTML"""
 
-    def run(self, md_string):
-        self.heading = 'I am a heading'
+    def run(self, markdown_string):
+        kordac_extension = KordacExtension()
+        kordac_extension.heading = None
         self.required_files = {}
-        self.html_string = ''
-        html = None
-        ext = KordacExtension()
+
         converter = markdown.Markdown(extensions=[
             'markdown.extensions.fenced_code',
             'markdown.extensions.codehilite',
             'markdown.extensions.sane_lists',
             mdx_math.MathExtension(enable_dollar_delimiter=True),
-            ext])
+            kordac_extension])
+        html = converter.convert(markdown_string)
+
+        result = KordacResult(
+            html=html,
+            heading=kordac_extension.page_heading
+        )
+        return result
+
 
-        self.html_string = converter.convert(md_string)
-        self.heading = ext.page_heading
+class KordacResult(object):
+    """Object created by Kordac containing result of
```

Comment:
Being super pedantic and asking that you are consistent in how you write your docstrings.
`"""docstring"""` or 
```
"""
docstring
"""
```
are fine, but just pick one, it's either inline or it's not :P